src/NzbDrone.Core/Tv/Movie.cs

```diff
@@ -54,10 +54,29 @@ public Movie()
 
         public bool HasFile => MovieFileId > 0;
 
-        public bool IsAvailable()
+        public bool IsAvailable(int delay = 0)
         {
-            //TODO might need to update this for preDB.
-            return (Status >= MinimumAvailability || (MinimumAvailability==MovieStatusType.PreDB && Status >= MovieStatusType.Released));
+            DateTime MinimumAvailabilityDate;
+            switch (MinimumAvailability)
+            {
+                case MovieStatusType.TBA:
+                case MovieStatusType.Announced:
+                    MinimumAvailabilityDate = DateTime.MinValue;
+                    break;
+                case MovieStatusType.InCinemas:
+                    if (InCinemas.HasValue)
+                        MinimumAvailabilityDate = InCinemas.Value;
+                    else
+                        MinimumAvailabilityDate = DateTime.MaxValue;
+                    break;
+                //TODO: might need to handle PreDB but for now treat the same as Released
+                case MovieStatusType.Released:
+                case MovieStatusType.PreDB:
+                default:
+                    MinimumAvailabilityDate = PhysicalRelease.HasValue ? PhysicalRelease.Value : (InCinemas.HasValue ? InCinemas.Value.AddDays(90) : DateTime.MaxValue);
+                    break;
+            }
+            return DateTime.Now >= MinimumAvailabilityDate.AddDays(delay);
         }
```

Comment:
@onedr0p : Problem is... I need to get the delay in here... somehow, which is from the Radarr Settings..